### DISCUSSION TITLE
I don't know what I'm doing...

### DISCUSSION THREAD

**Discussion by Imanol82** [john30/ebusd#1282] [Q&A]
Hello.

First question because I started installing things and I forgot the most important question.

Is this amazing sieldh v5 compatible with my Baxi Aerotermia?
Right now a Baxi TXM 10 thermostat is connected via ebus to it (brand is Remeha). And I would like to remove it because it is garbage, and control everything from HA.

Let's first see if it's compatible before we continue messing around...

Thank you so much!

**Comment by JonesGW-MGD** [comment]
If the Baxi Aerotermia is ebus-enabled then in theory, yes.

You'll find however that the latest supported configuration files are exclusively Vaillant products.  Earlier file-sets included limited support for Ochsner, Protherm and Wolf and other users have hand-rolled their own files for other ebus-using manufacturers.  It's possible someone has tried with your brand - you'd have to google, but on this forum, your's is the only post that mentions Baxi I think!

The configuration files are essentially what maps and decodes the ebus code from the sending or receiving addresses.  Otherwise, the traffic is simply an unintelligible series of hex 'telegrams' or messages.

So it likely means you'll need to get heavily involved in interpreting the ebus traffic and decoding the messages yourself - which is not for the faint hearted!

(Are you sure Baxi is ebus?  In the UK Baxis are OpenTherm and not ebus and the TXM-10C connects via ITM-10C (Which is OpenTherm protocol.  But it may differ in your territory)

**Reply by Imanol82** [reply]
I don't know my friend. I'm looking for info on Google and nothing...

This is my equipment, the middle one:
![Screenshot_2024-05-31-22-09-36-174_com google android apps docs](https://github.com/john30/ebusd/assets/109697268/dc8d3b1c-f3d3-4b2b-a5dd-6105c84110f0)

And these are the boards. With Rbus connector:
![Screenshot_2024-05-31-22-09-10-425_com google android apps docs](https://github.com/john30/ebusd/assets/109697268/7736989b-654f-461d-b53f-01fb9f97e7bc)

Any idea?

Thank you very much!!

**Reply by JonesGW-MGD** [reply]
I don't really know for certain - looks like R-Bus is 24v, like ebus but I don't think it **is** ebus (my guess) so in which case the shield won't work.  Most Baxi installs I've seen are straight system gas boilers or combination boilers and they have a TXM-10 which connects to the boiler via GTW-16 gateway that converts Opentherm (OT) from boiler to RBUS for thermostat, but confusingly, your boiler has RBUS connectors directly on the PCB.  

My schoolboy Spanish however indicates that X12 / X3 says it's RBUS **or** OT?

Either way, whether through a RBUS to OT converter or OT natively you should be able to control your device via OT, for which there are integrations to Home Assistant, rather than use ebus - like I say, not certain however.

You may still find yourself needing to decode the traffic yourself however unless you find someone with a similar install.  I don't know whether this link helps or not but folks are discussing Baxi control via OT in HA:

https://community.home-assistant.io/t/baxi-txm-rxm-10c-opentherm-thermostat-ha-integration/358612/55?page=2

**Reply by Imanol82** [reply]
Thank you very much for your support.
I have all these doubts too haha
I'll try different solutions. 

Thank you!
